chapters/my/chapter1/1.mdx

```diff
@@ -0,0 +1,105 @@
+
+[**Lewis Tunstall**](https://huggingface.co/lewtun) သည် Hugging Face တွင် machine learning engineer တစ်ဦးဖြစ်ပြီး open-source tools များကို တီထွင်ခြင်းနှင့် ပိုမိုကျယ်ပြန့်သော အသိုင်းအဝိုင်းအတွက် လွယ်ကူစွာ အသုံးပြုနိုင်စေရန် အာရုံစိုက်သည်။ သူသည် O'Reilly စာအုပ် [Natural Language Processing with Transformers](https://www.oreilly.com/library/view/natural-language-processing/9781098136789/) ၏ ပူးတွဲရေးသားသူလည်း ဖြစ်သည်။
+
+[**Leandro von Werra**](https://huggingface.co/lvwerra) သည် Hugging Face ရှိ open-source အဖွဲ့တွင် machine learning engineer တစ်ဦးဖြစ်ပြီး O'Reilly စာအုပ် [Natural Language Processing with Transformers](https://www.oreilly.com/library/view/natural-language-processing/9781098136789/) ၏ ပူးတွဲရေးသားသူလည်းဖြစ်သည်။ Machine learning stack တစ်ခုလုံးတွင် လုပ်ကိုင်ခဲ့ခြင်းဖြင့် NLP ပရောဂျက်များကို ထုတ်လုပ်မှုအဆင့်သို့ ရောက်ရှိစေရန် နှစ်ပေါင်းများစွာ အတွေ့အကြုံရှိသည်။
+
+## မေးလေ့ရှိသောမေးခွန်းများ
+
+မေးလေ့ရှိသောမေးခွန်းများအတွက် အဖြေအချို့ကို ဤနေရာတွင် ဖော်ပြထားပါသည်။
+
+- **ဤသင်တန်းကို တက်ရောက်ခြင်းဖြင့် အသိအမှတ်ပြုလက်မှတ် ရရှိပါသလား။**
+လောလောဆယ်တွင် ဤသင်တန်းအတွက် အသိအမှတ်ပြုလက်မှတ် မရှိသေးပါ။ သို့သော် Hugging Face ecosystem အတွက် အသိအမှတ်ပြုလက်မှတ်ပရိုဂရမ်တစ်ခုကို ကျွန်ုပ်တို့ လုပ်ဆောင်နေပါသည် - စောင့်မျှော်ကြည့်ရှုပါ။
+
+- **ဤသင်တန်းတွင် အချိန်မည်မျှ သုံးစွဲသင့်သနည်း။**
+ဤသင်တန်းရှိ အခန်းတစ်ခန်းစီကို တစ်ပတ်အတွင်း ပြီးမြောက်အောင် ဒီဇိုင်းထုတ်ထားပြီး တစ်ပတ်လျှင် ၆-၈ နာရီခန့် လုပ်ဆောင်ရန် လိုအပ်ပါသည်။ သို့သော် သင်တန်းပြီးဆုံးရန် လိုအပ်သလောက် အချိန်ယူနိုင်ပါသည်။
+
+- **မေးခွန်းတစ်ခုခုရှိရင် ဘယ်မှာမေးလို့ရမလဲ။**
+သင်တန်း၏ အပိုင်းတစ်ခုခုနှင့်ပတ်သက်၍ မေးခွန်းတစ်ခုခုရှိပါက စာမျက်နှာ၏ ထိပ်ရှိ "*Ask a question*" banner ကို နှိပ်ပါက [Hugging Face forums](https://discuss.huggingface.co/) ၏ သက်ဆိုင်ရာ အပိုင်းသို့ အလိုအလျောက် ပြန်ညွှန်းပေးပါမည်။
+
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter1/forum-button.png" alt="Hugging   
+ Face forums သို့ လင့်ခ်" width="75%">
+
+သင်တန်းပြီးဆုံးပါက ပိုမိုလေ့ကျင့်လိုပါက [ပရောဂျက်အကြံဥာဏ်များ](https://discuss.huggingface.co/c/course/course-event/25) စာရင်းကိုလည်း forum များတွင် ရရှိနိုင်ကြောင်း သတိပြုပါ။
+
+- **သင်တန်းအတွက် ကုဒ်ကို ဘယ်မှာရနိုင်မလဲ။**
+အပိုင်းတစ်ခုစီအတွက် စာမျက်နှာ၏ ထိပ်ရှိ banner ကို နှိပ်ပါက Google Colab သို့မဟုတ် Amazon SageMaker Studio Lab တွင် ကုဒ်ကို လုပ်ဆောင်နိုင်ပါသည်။
+
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter1/notebook-buttons.png" alt="Hugging   
+ Face သင်တန်း notebooks များသို့ လင့်ခ်" width="75%">
+
+သင်တန်းမှ ကုဒ်အားလုံးပါဝင်သော Jupyter notebooks များကို [`huggingface/notebooks`](https://github.com/huggingface/notebooks) repo တွင် ထားရှိပါသည်။ ၎င်းတို့ကို ဒေသတွင်းတွင် generate လုပ်လိုပါက GitHub ရှိ [`course`](https://github.com/huggingface/course#-jupyter-notebooks) repo ရှိ ညွှန်ကြားချက်များကို ကြည့်ရှုပါ။
+
+- **ဤသင်တန်းကို ပြန်လည်အသုံးပြုနိုင်ပါသလား။**

```

Comment:
I think we're missing two points here:

- How can I contribute to the course?
- What were the choices made for each translation?